ctcdecode/__init__.py

```diff
@@ -26,12 +26,12 @@ def __init__(
     def decode(self, probs, seq_lens=None):
```

Comment:
I'm still expecting torch tensor here so that we can keep other places unchanged.